examples/text-generation/run_lm_eval.py

```diff
@@ -60,6 +80,8 @@ def setup_lm_eval_parser():
         default=["hellaswag", "lambada_openai", "piqa", "winogrande"],
     )
     parser.add_argument("--limit_iters", type=int, help="limit examples to run that many iterations", default=None)
+    parser.add_argument('--dont-trust-remote-datasets-code', action='store_false', default=True,
+                        dest='trust_remote_datasets_code', help='Trust remote code')
```

Comment:
```suggestion
```

@polisettyvarma, you do not need this, you can use https://github.com/huggingface/optimum-habana/blob/c495f479d9abf04fb7adb6f0a5607d7963186649/examples/text-generation/run_generation.py#L285